src/components/mortgage-investment-compare/amortization-table.js

```diff
@@ -133,8 +133,28 @@ const hoverTableCells = (year, option1, option2) => {
 const AmortizationTable = props => {
   return (
     <div>
-      <h4>Yearly Breakdown</h4>
-      <Table bordered responsive className='mt-3'>
+      <div className='d-flex flex-row align-items-baseline'>
+        <h4>Yearly Breakdown</h4>
+        <CustomInput
+          type='checkbox'
+          className='custom-switch ml-4'
+          id='roi-switch-yearly-breakdown'
+          name='roi-switch-yearly-breakdown'
+          label={<small>Include ROI?</small>}
+          checked={props.includeROI}
+          onChange={props.onROISwitch}
+        />
+        <CustomInput
+          type='checkbox'
+          className='custom-switch ml-4'
+          id='inflation-switch-yearly-breakdown'
+          name='inflation-switch-yearly-breakdown'
+          label={<small>Include Inflation?</small>}
+          checked={props.includeInflation}
+          onChange={props.onInflationSwitch}
+        />
```

Comment:
should the switches be light blue like our links and feedback button?

Reply:
if possible but nbd

I think in Sass we would have defined our standard colors w/ some variables and that should make all colors consistent to our theme. Not sure how to do that in pure CSS

Reply:
darnit david, just saw this comment. Making a new PR

Reply:
nah let's call it

Reply:
NO I MUST REACH PERFECTION

Reply:
just kidding, looked into it, it's a little too much work for the benefit